x-pack/test/apm_api_integration/tests/services/throughput.ts

```diff
@@ -53,31 +61,96 @@ export default function ApiTest({ getService }: FtrProviderContext) {
       });
 
       it('returns some data', () => {
-        expect(throughputResponse.throughput.length).to.be.greaterThan(0);
+        expect(throughputResponse.currentPeriod.length).to.be.greaterThan(0);
+        expect(throughputResponse.previousPeriod.length).not.to.be.greaterThan(0);
 
-        const nonNullDataPoints = throughputResponse.throughput.filter(({ y }) => y !== null);
+        const nonNullDataPoints = throughputResponse.currentPeriod.filter(removeInvalidYAxis);
```

Comment:
Honestly, I prefer the original version. Much easier to read and the duplication is not a problem

Reply:
@sqren  you mean to keep the `({ y }) => y !== null` on both arrays? I ask this because I created a utils file on another PR and added some functions that are used in more places, and one is this. https://github.com/elastic/kibana/pull/88946/files#diff-338cceb311b19305c8b57c7c2f88391066d018907737be9507fbe45c602ffd58R16

Reply:
Yeah, personally I think `({ y }) => y !== null` is more clear. I'd only extract to a util function if it's needed for narrowing the return type, like we do with `isFiniteNumber`:
https://github.com/elastic/kibana/blob/810e4ab8e8206949965c89c889aac2fc396c4111/x-pack/plugins/apm/common/utils/is_finite_number.ts#L11-L13

Reply:
Alright, I'm removing the function, and I'm going to remove it from the utils file in the other PR.